test/reporter-html/diff.js

```diff
@@ -0,0 +1,163 @@
+QUnit.module( "diff" );
+
+QUnit.test( "throws if arguments are not strings", function( assert ) {
+	assert.throws(function() { QUnit.diff( {}, "" ); });
+	assert.throws(function() { QUnit.diff( "", {} ); });
```

Comment:
I think these two are enough to verify that there is some argument checking in place, no need for further tests.